packages/expo-errors/src/Errors.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * A general error class that should be used for all errors in Expo modules.
```

Comment:
BTW if you use VS Code, there's a plugin called Rewrap that lets you hit Option-Q and it will flow text to match the set column limit - https://github.com/expo/expo/commit/190df5fb87b44fb1503c77d04bee4a1e539c465e